### ISSUE TITLE
Quickblox android Sdk Crash

### ISSUE THREAD

**Issue by lazy-coder-10** [QuickBlox/quickblox-android-sdk#754]
**Yesterday I have got crashes in my application but when I checked the Crashlytics dashboard that was because of QB. QB was getting crash again and again for 1 or 2 hours and then automatically got fixed**
 
*(Operating system, browser information, SDK version)*
QBSdkVersion = 3.9.2
Android version = 6.0.1

**Did this work before?**
Yes, it was working fine earlier.

**Logs**
*Caused by java.lang.IllegalStateException: Expected BEGIN_ARRAY but was BEGIN_OBJECT at line 1 column 12 path $.errors
       at com.google.gson.stream.JsonReader.beginArray(JsonReader.java:350)
       at com.google.gson.internal.bind.CollectionTypeAdapterFactory$Adapter.read(CollectionTypeAdapterFactory.java:80)
       at com.google.gson.internal.bind.CollectionTypeAdapterFactory$Adapter.read(CollectionTypeAdapterFactory.java:61)
       at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$1.read(ReflectiveTypeAdapterFactory.java:131)
       at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:222)
       at com.google.gson.Gson.fromJson(Gson.java:927)
       at com.google.gson.Gson.fromJson(Gson.java:892)
       at com.google.gson.Gson.fromJson(Gson.java:841)
       at com.google.gson.Gson.fromJson(Gson.java:813)
       at com.quickblox.core.parser.QBJsonErrorParser.parseError(:67)
       at com.quickblox.core.result.RestResult.parseErrors(:4)
       at com.quickblox.core.result.RestResult.processResponse(:16)
       at com.quickblox.core.result.RestResult.setResponse(:25)
       at com.quickblox.core.parser.QBJsonParser.parse(:21)
       at com.quickblox.auth.session.Query.parseAccountRequest(:20)
       at com.quickblox.auth.session.Query.access$000()
       at com.quickblox.auth.session.Query$1.completedWithResponse(:9)
       at com.quickblox.core.server.HttpRequestRunnable$1.handleMessage(:48)
       at android.os.Handler.dispatchMessage(Handler.java:107)
       at android.os.Looper.loop(Looper.java:213)
       at android.app.ActivityThread.main(ActivityThread.java:8178)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:513)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1101)*
       
       Another Log ::
       *Fatal Exception: java.lang.RuntimeException: An error occurred while executing doInBackground()
       at android.os.AsyncTask$3.done(AsyncTask.java:309)
       at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:354)
       at java.util.concurrent.FutureTask.setException(FutureTask.java:223)
       at java.util.concurrent.FutureTask.run(FutureTask.java:242)
       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1113)
       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:588)
       at java.lang.Thread.run(Thread.java:818)*

**Comment by alexparvus1** [comment]
@rohit6027 Please provide more detailed logs

**Comment by lazy-coder-10** [comment]
It happens only once.

**Comment by QB-nikolay-zolotarov** [comment]
@lazy-coder-10 
Hello,

This is Nikolay from QuickBlox support.

Please let me know if the issue is still relevant.

If there is no reply within 5 business days, the issue will be closed as outdated.

**Comment by QB-nikolay-zolotarov** [comment]
@lazy-coder-10 The issue is being closed as outdated.

If it is still relevant, feel free to open a new issue.
